.vscode/launch.json

```diff
@@ -0,0 +1,30 @@
+{
+    // Use IntelliSense to learn about possible attributes.

```

Comment:
I guess VSCode is smart enough not to choke on comments?  Typically this would fail to parse.

Reply:
Yeah this is included by default in the built-in launch.json template.